gradle-modules/src/main/kotlin/org.sonarsource.cloud-native.artifactory-configuration.gradle.kts

```diff
@@ -15,73 +15,22 @@
  * along with this program; if not, see https://sonarsource.com/license/ssal/
  */
 import org.sonarsource.cloudnative.gradle.ArtifactoryConfiguration
-import org.sonarsource.cloudnative.gradle.signingCondition
 
 plugins {
-    id("com.jfrog.artifactory")
-    signing
+    // `maven-publish` is required for the `artifactory` plugin (or the root project shoudn't be published)
     `maven-publish`
+    id("com.jfrog.artifactory")
+}
+
+if (project.path != ":") {
+    throw GradleException("This build script must be applied to the root project only")
 }
 
 // this value is present on CI
 val buildNumber: String? = System.getProperty("buildNumber")
-if (project.version.toString().endsWith("-SNAPSHOT") && buildNumber != null) {
-    val versionSuffix = if (project.version.toString().count { it == '.' } == 1) ".0.$buildNumber" else ".$buildNumber"
-    project.version = project.version.toString().replace("-SNAPSHOT", versionSuffix)
-    logger.lifecycle("Project version set to $version")
-}
```

Comment:
It is interesting that this logic is not needed anymore

Reply:
Yes, it's already applied by the settings plugin